opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -201,6 +201,16 @@ const fn severity_of_level(level: &Level) -> Severity {
     }
 }
 
+const fn severity_text_of_level(level: &Level) -> &'static str {
+    match *level {
+        Level::TRACE => "TRACE",
```

Comment:
`Severity::Warn.name()` could have been used as well, but I think this would be the fastest. I believe compiler is smart enough to avoid storing the string duplicates...

Reply:
We can get rid of this method. `meta.level().as_str()` does exactly this.

https://github.com/tokio-rs/tracing/blob/master/tracing-core/src/metadata.rs#L524-L535

Reply:
🤣 I am considering removing populating severitytext at all, and let exporter figure it out from the severitynumber.

Reply:
Haha okay. Let's do that right away or remove this method until make the change to stop populating severity text?